packages/gatsby-theme-docs/src/hooks/use-all-release-notes.js

```diff
@@ -0,0 +1,40 @@
+import { useStaticQuery, graphql } from 'gatsby';
+
+const useAllReleaseNotes = () => {
+  const queryResult = useStaticQuery(
+    graphql`
+      query GetAllReleaseNotesForFilters {
+        allReleaseNotePage(sort: { order: DESC, fields: date }) {
+          nodes {
+            date(formatString: "YYYY-MM-DD")
+            topics
+          }
+        }
+      }
+    `
+  );
+
+  return queryResult.allReleaseNotePage.nodes;
+};
+
+export default useAllReleaseNotes;
```

Comment:
Use a named export for this as well, since you also have other exports (not just one default).